index.bs

```diff
@@ -2758,44 +2793,69 @@ Appeal by Advisory Committee Representatives</h3>
+	on the proposal to overturn the decision.
+
+	The conclusion of the [=Advisory Committee Override=] is final:
+	the same decision cannot be appealed more than once.
+
+	When the [=Advisory Committee Override=]
+	approves the proposal to overturn the decision,
+	the proposal <em class=rfc2119>may</em> be returned for additional work,
+	after which the applicable decision process is re-initiated.
+
+<h3 id="ac-override">
+Advisory Committee Override</h3>
+
+	An <dfn export>Advisory Committee Override</dfn> is an exceptional two-step procedure
+	used to resolve certain matters where the usual [=consensus=] process
+	and its [[#addressing-fo|escalation path]] are insufficient.
+
+	Note: Currently, this only applies to [=AC Appeals=]
+	and [=recalls=] of [=AB=] or [=TAG=] participants.
+
+	First, the [=Team=] <em class="rfc2119">must</em> announce the proposal for a [=Advisory Committee Override=]
 	to the [=Advisory Committee=]
 	and provide a mechanism for [=Advisory Committee representatives=]
-	to respond with a statement of positive support for this appeal.
+	to respond with a statement of positive support for holding a vote.
 	The archive of these statements <em class="rfc2119">must</em> be [=member-only=].
+
 	If, within <span class="time-interval">one week</span> of the Team's announcement,
-	5% or more of the [=Advisory Committee=] support the appeal request,
-	the Team <em class="rfc2119">must</em> organize an appeal vote
+	5% or more of the [=Advisory Committee=] support the holding the vote,
+	the Team <em class="rfc2119">must</em> organize a vote
 	asking the [=Advisory Committee=]
-	“Do you approve of the Decision?”
-	together with links to the decision and the appeal support.
+	“Do you approve of the Advisory Committee Override proposal?”
+	together with details about the proposal
+	and links to support for holding the vote.
 
 	The ballot <em class="rfc2119">must</em> allow for three possible responses:
 	“Approve”,
 	“Reject”,
 	and “Abstain”,
 	together with Comments.
 
-	If the number of votes to reject
-	exceeds the number of votes to approve,
-	the decision is overturned.
-	In that case, there are the following possible next steps:
-
-	<ol>
-		<li>
-			The proposal is rejected.
-
-		<li>
-			The proposal is returned for additional work,
-			after which the applicable decision process is re-initiated.
-	</ol>
+	The level of support needed for a [=Advisory Committee Override=] to pass
+	depends on the level of participation by [=Advisory Committee Representatives=],
+	(including those who cast an explicit “abstain” ballot):
+	* if no more than 15%,
+		the vote passes
+		if the number of “Approve” ballots exceeds three times the number of “Reject” ballots.
+	* if more than 15% but less than 20%,
+		the vote passes
+		if the number of “Approve” ballots exceeds twice the number of “Reject” ballots.
+	* if 20% or more,
+		the vote passes
+		if the number of “Approve” ballots exceeds the number of “Reject” ballots.
```

Comment:
```suggestion
	* if 15% or fewer participate,
		and the number of “Approve” ballots exceeds triple (3x) the number of “Reject” ballots,
		the vote passes.
	* if more than 15% but fewer than 20% participate,
		and the number of “Approve” ballots exceeds double (2x) the number of “Reject” ballots,
		the vote passes.
	* if 20% or more participate,
		and the number of “Approve” ballots exceeds the number of “Reject” ballots,
		the vote passes.
```

Reply:
I think I like the rephrasing in terms of the first line of each bullet, but I prefer the original for the rest of the text. Any third opinion?

Reply:
"If, then, if" construction — i.e., having the "if" portions of each bullet broken by the "then" portion — makes little sense to me. I feel strongly that the simpler "if, and, then" construction is clearer. (All "then" are implicit, and I think they do not need to be made explicit.)

Reply:
Ok, I think I agree with this latest point of yours. I'd be in favor of taking in your overall wording, but the one part I a still not convinced about is replacing
> the number of “Approve” ballots exceeds _twice_ the number of “Reject” ballots

with

> the number of “Approve” ballots exceeds _double (2x)_ the number of “Reject” ballots

I think both are correct, but the first one sounds better to my ear. But then again, I'm not a native speaker. Am I wrong here?

Reply:
'twice' sounds better to me but I am not sure why. (and 'thrice' would be the matching archaic word)

Reply:
`Twice` and `thrice` are certainly options. But I think both are more likely to be misunderstood by non-native English readers than `double` and `triple`.

I added the 2x and 3x parentheticals to help clarity for those non-native English readers.

A mid-ground might be `twice (2x)` and `thrice (3x)`.

Reply:
I would not use the word "thrice", it's considered somewhat archaic. Twice (2x) and three times (3x) would be fine by me.

Reply:
`Twice (2x)` and `three times (3x)` applied to my change suggestion.